pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -330,30 +332,19 @@ func (l *podNIC) getPhase2Binding(domain *api.Domain) (BindMechanism, error) {
 }
 
 func newMacvtapBindMechanism(vmi *v1.VirtualMachineInstance, iface *v1.Interface, domain *api.Domain, cacheFactory cache.InterfaceCacheFactory, launcherPID *int, handler netdriver.NetworkHandler) (*MacvtapBindMechanism, error) {
```

Comment:
there's no longer any reason for these constructors to return an error (since we no longer read the MAC addr from the spec).